windows/CodePush.Shared/CodePushUtils.cs

```diff
@@ -61,13 +60,10 @@ internal static string GetAssetsBundlePrefix()
 #endif
         }
 
-        internal static string GetFileBundlePrefix()
+        internal static string ExtractSubFolder(string fullPath)
         {
-#if WINDOWS_UWP
-            return CodePushConstants.FileBundlePrefix;
-#else
-            return GetAppFolder();
-#endif
+            var codePushSubPathArray = fullPath.Split(Path.DirectorySeparatorChar);
+            return String.Join("/", codePushSubPathArray.SkipWhile((value, index) => codePushSubPathArray.Length - index > 4).ToArray());
```

Comment:
4 means last 4 segments of the path where JS bundle is located. The idea of the change was to unify the calculation of js bundle file location for UWP and WPF. While they have different prefix but last 4 segments are the same for both implementations.
For example for
 `F:\react-native-code-push\Examples\CodePushDemoApp\windows\CodePushDemoApp.Wpf\bin\x64\DebugBundle\CodePush\cc1dd1ab\CodePush\index.windows.bundle`
the part that will be added to specific prefix will be:
`CodePush/cc1dd1ab/CodePush/index.windows.bundle`